contracts/ManagerProxy.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.13;
+
+import "./ManagerProxyTarget.sol";
+
+
+contract ManagerProxy is ManagerProxyTarget {
+    function ManagerProxy(address _controller, bytes32 _targetContractId) Manager(_controller) {
+        targetContractId = _targetContractId;
+    }
+
+    // Based on https://github.com/AugurProject/augur-core/blob/develop/src/libraries/Delegator.sol
+    function() public payable {
```

Comment:
Dose this make every call payable?

Reply:
Discussed offline and the fallback function needs to be payable for the case when a user is calling the `verify()` function in JobsManager which is payable. If we really wanted to prevent anyone from accidentally sending ether to this contract except for when `verify()` is called, we could add a check for the method signature and if it is not the signature for `verify()` and `msg.value > 0` we throw